### PR TITLE
Changed healths endpoint port

### PR DESCRIPTION
**Pull request by olsavchenk**

### CODE REVIEW

**Review by VadymLevkovskyi — COMMENTED** [review]
I'm curious why approach suggested in [SO discussion](https://stackoverflow.com/questions/57721900/healthchecks-on-different-port-in-net-core-web-application) doesn't work

File: OutOfSchool/OutOfSchool.WebApi/Startup.cs [hunk 1]
```diff
@@ -93,6 +91,7 @@ public static void Configure(this WebApplication app)
                 Predicate = healthCheck => healthCheck.Tags.Contains("readiness"),
                 AllowCachingResponses = false,
             })
+            .RequireHost("*:9000")
```
Comment (VadymLevkovskyi) [inline OutOfSchool/OutOfSchool.WebApi/Startup.cs]: (Code style) Magic number, that probably has to be `ApplicationPorts:HealthPort`.

Comment (olsavchenk) [reply OutOfSchool/OutOfSchool.WebApi/Startup.cs]: Somehow missed that, it's fixed now

**Review by VadymLevkovskyi — APPROVED** [review]
Approved, though suggested middleware seems excessive
At same time I have no more resources than mentioned SO discussion to prove that it could be simplified, so let it be.

**Pull request merged**